contributor/reference/stable-release.rst

```diff
@@ -0,0 +1,76 @@
+Stable release process
+======================
+
+Key Steps at Each Stage
+-----------------------
+
+Rocks
++++++
+
+Currently, the rocks are published directly to the registry when a modification is made.
+The Rocks mono-repository is maintains a stable branch from which new rocks are built and published.
+
+The process is as follows:
```

Comment:
Add more description, e.g. "Publishing rocks to the registry follows the following steps:
* **Backport**: ...
* **Approval**: ...
* **Build:** ...
Is there any reason not to capitalise these steps?